lib/astarte_data_access/mappings.ex

```diff
@@ -21,12 +21,19 @@ defmodule Astarte.DataAccess.Mappings do
   alias CQEx.Query
   require Logger
 
-  @spec fetch_interface_mappings(:cqerl.client(), binary) ::
+  @spec fetch_interface_mappings(:cqerl.client(), binary, keyword) ::
           {:ok, list(%Mapping{})} | {:error, atom}
-  def fetch_interface_mappings(db_client, interface_id) do
+  def fetch_interface_mappings(db_client, interface_id, opts \\ []) do
+    get_docs_statement =
+      case Keyword.get(opts, :get_docs) do
+        true -> ", doc, description"
+        _ -> ""
+      end
```

Comment:
1. I'd call the option `:include_docs`
2. You can use `if` instead of case, since the default is `nil` which is falsey